spec/linter-pydocstyle-spec.coffee

```diff
@@ -1,4 +1,4 @@
-LinterPEP257 = require '../lib/init'
+LinterPydocstyle = require '../lib/init'
```

Comment:
It's just a local reference, so it's however you want it to look. Nothing external to this file uses the var.